src/races.cpp

```diff
@@ -1381,61 +1381,46 @@ void init_races(void) {
             ADD_SKILL(SKILL_BREATHE_LIGHTNING, ROLL_SKILL_PROF);
             ADD_SKILL(SKILL_SWEEP, ROLL_SKILL_PROF);
             ADD_SKILL(SKILL_ROAR, ROLL_SKILL_PROF);
-            ADD_SKILL(SPELL_ACID_BREATH, 1000);
```

Comment:
How do we assign these skills now?

Reply:
ADD_SKILL(SKILL_BREATHE_LIGHTNING, ROLL_SKILL_PROF);

This line is all we need to give characters the skill, which calls the spell regardless of if the player "knows" the spell.

Since the spell isn't being called through the "cast" command, it doesn't matter what sphere or skill the spell usually uses.